### DISCUSSION TITLE
make error in docker image

### DISCUSSION THREAD

**Discussion by antoineschutz** [prosyslab-classroom/cs348-information-security#750] [Q&A]
Name: Antoine Schutz 

Hello, I'm having troubles to run make in the docker image . I think i've followed the Environment setup instruction , but when I try to use the command make  , I get this error : 
"make: dune: Command not found
make: *** [Makefile:7: all] Error 127"

I have followed the "[Get Up and Running With OCaml](https://ocaml.org/docs/up-and-running)" documentation of OCAML.org on my own machine and make works perfectly , but I cannot make it work on docker.

**Comment by KAIST-JongchanPark** [comment]
Would you try `eval $(opam env)`?

**Comment by antoineschutz** [comment]
Update the issue was cause by a mistake in how I cloned the repo inside docker , everythin works well now . Thank you !
